### PR TITLE
Member services

### PR DESCRIPTION
**Pull request by queenieho**
• has-more? subscription implemented for catalogues
• updated shopping cart button to update cost and item count when new items are added
• added route to shopping cart page

### CODE REVIEW

File: src/cljs/member/services/views.cljs [hunk 1]
```diff
@@ -167,6 +170,7 @@
         c          (first (filter #(= @selected (:key %)) @catalogues))]
```
Comment (jalehman) [inline src/cljs/member/services/views.cljs]: `c` isn't actually used anywhere; let's remove this line and the corresponding `log` below on line 173.

File: src/cljs/member/services/views.cljs [hunk 2]
```diff
@@ -85,21 +85,24 @@
...
-       [:div.column.is-7
-        [:h4.subtitle.is-5 "X services being requested"]]
-       [:div.column.is-1
-        [:p.price (format/currency 150.0)]]
-       [:div.column.is-4.has-text-right
-        [ant/button "Submit Request"]]]]])
+  (let [item-count (subscribe [:services.cart/item-count])
+        total-cost (subscribe [:services.cart/total-cost])]
+    [ant/affix {:offsetBottom 20}
+     (.log js/console @item-count)
```
Comment (jalehman) [inline src/cljs/member/services/views.cljs]: remove

File: src/cljs/member/services/subs.cljs [hunk 3]
```diff
@@ -113,3 +113,30 @@
...
+   (count db)))
+
+;; NOTE when testing be aware that some items and their respective service ids are repeated in
+;;      test data. If we test adding those, the cost of every repeat will be added to the total cost
+(reg-sub
+ :services.cart/total-cost
+ :<- [:services.book/catalogues]
+ :<- [:services.cart/cart]
+ (fn [[catalogues cart] _]
+   (.log js/console catalogues)
```
Comment (jalehman) [inline src/cljs/member/services/subs.cljs]: remove

File: src/cljs/member/services/views.cljs (hunk 1 above)
Comment (queenieho) [reply src/cljs/member/services/views.cljs]: it actually is, we use it when a catagory other than "all" is selected

Comment (jalehman) [reply src/cljs/member/services/views.cljs]: Ah, I see now.

**Pull request closed without merging**